projects/manager/src/app/datasets/dataset-editor/dataset-editor.component.html

```diff
@@ -31,6 +31,25 @@
                         <mat-label>Dataset Description</mat-label>
                         <textarea matInput formControlName="description"></textarea>
                     </mat-form-field>
+                    <mat-form-field>
+                        <mat-label>Tags</mat-label>
+                        <mat-chip-grid #tagsGrid formControlName="tags">
+                            <mat-chip-row *ngFor="let column of form.controls.tags.value" (removed)="removeTag(column)">
+                                {{ column }}
+                                <mat-icon matChipRemove>cancel</mat-icon>
+                            </mat-chip-row>
+                            <input
+                                formControlName="newTag"
+                                placeholder="New tag..."
+                                [matChipInputFor]="tagsGrid"
+                                (matChipInputTokenEnd)="addTag($event)"
+                            />
+                            <mat-error *ngIf="tagInputControl.errors?.invalidTag">
+                                Error: Invalid tag. Space, / and ".." are not allowed.
+                            </mat-error>
+                            <mat-error *ngIf="tagsControl.errors?.duplicate"> Error: Duplicate tag. </mat-error>
```

Comment:
Isn't it clear that a red text indicates an error?
I would remove the prefix.

```suggestion
                            <mat-error *ngIf="tagInputControl.errors?.invalidTag">
                                Invalid tag. Space, / and ".." are not allowed.
                            </mat-error>
                            <mat-error *ngIf="tagsControl.errors?.duplicate">Duplicate tag.</mat-error>
```